web/src/components/DandisetsPage.vue

```diff
@@ -51,7 +51,10 @@
         >
           {{ option.name }}
           <v-icon right>
-            <template v-if="sortDir === 1 || sortOption !== i">
+            <!-- For some reason, MDI's icons are represented backwards of what you'd expect.
+             Descending has an arrow pointing upwards, which is usually meant to represent ascending.
+             We intentionally switch the values here, this is not a bug. -->
```

Comment:
Oh goodness, I think I figured it out. Note that the icons both have three bars of different lengths representing data items. The lines get longer as you move down... so "ascending" is an arrow pointing down (towards the longer lines) while "descending" (of course) points up. That seems like bad icon design to me.

We could use the "wrong" icons as in this PR, or we could consider using these two ([ascending](https://pictogrammers.com/library/mdi/icon/sort-variant/), [descending](https://pictogrammers.com/library/mdi/icon/sort-reverse-variant/)) that don't feature the "backward" arrow. (I am leaning towards these "variant" icons myself, because using the "bad" icons, and putting the "bad" icons "backwards" both feel like bad options.)

Reply:
Well I've changed the icons to your suggestion, as I think it does improve readability, although notably they are still backwards. `sort-variant`, which "should" correspond to ascending sort, shows the largest item at the top. However, visually this would correspond with descending sort. You can check out the deploy preview to see it, IMO I don't see many people preferring it the other way around.